### PR TITLE
fix: incorrect color used for Margin

### PR DESCRIPTION
**Pull request by ssbarbee**
bug reported here: https://discord.com/channels/896685581487210577/1005057197509709945/1034553453093060618

also fixed an issue when the value is 0.00 and we show "-0.00" ^_^ (notice the -)

### CODE REVIEW

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx [hunk 1]
```diff
@@ -40,22 +39,34 @@ const CurrentMargin: React.FunctionComponent<CurrentMarginProps> = ({
...
   };
 
-  const getNetMarginLabel = () => (
-    <>
-      Margin
-      {!isUndefined(accruedCashflow) && !isSettled && (
-        <Box
-          component="span"
-          sx={{ color: accruedCashflow >= 0 ? colors.vzCustomGreen1 : colors.vzCustomRed1 }}
+  const getNetMarginLabel = () => {
```
Comment (ioanaroceanu2000) [inline src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:42]: should we have this as an useMemo?

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx [hunk 2]
```diff
@@ -40,22 +39,34 @@ const CurrentMargin: React.FunctionComponent<CurrentMarginProps> = ({
...
+      <>
+        Margin
+        <span
+          style={{
+            color:
+              accFlowValue > 0
+                ? colors.vzCustomGreen1.base
+                : accFlowValue < 0
+                ? colors.vzCustomRed1.base
+                : undefined,
```
Comment (ioanaroceanu2000) [inline src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:60]: /Users/ioanaroceanu/Documents/Voltz/Voltz-Leaderboard/config

Comment (ioanaroceanu2000) [inline src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:60]: what's the default color?

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx (hunk 1 above)
Comment (ssbarbee) [reply src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:42]: no need, this is not heavy

File: src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx (hunk 2 above)
Comment (ssbarbee) [reply src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx:60]: whatever the parent component will provide, hence undefined here. To respect that style

**Review by ioanaroceanu2000 — APPROVED** [review]

**Pull request merged**